src/render.ts

```diff
@@ -0,0 +1,332 @@
+ <path d="M10851 2187 c-49 -29 -66 -101 -35 -146 9 -13 32 -29 50 -37 29 -12 39 -12 68 0 99 41 85 180 -19 192 -24 3 -50 -1 -64 -9z"/>
+ <path d="M10660 2136 c-19 -23 -8 -61 18 -64 44 -7 67 32 36 62 -19 20 -38 20 -54 2z"/>
+ <path d="M11096 2124 c-9 -8 -16 -22 -16 -29 0 -13 26 -45 36 -45 20 0 44 25 44 45 0 14 -8 29 -19 35 -25 13 -27 13 -45 -6z"/>
+ <path d="M10335 1991 c-60 -6 -60 -6 -57 -36 9 -69 104 -248 122 -229 57 61 210 250 207 258 -4 12 -176 17 -272 7z"/>
+ <path d="M11267 1983 c-68 -5 -79 -19 -47 -60 23 -31 200 -193 210 -193 3 0 20 24 37 53 29 48 52 111 67 180 l6 27 -107 -2 c-60 -1 -134 -3 -166 -5z"/>
+ <path d="M10870 1910 c-16 -31 4 -62 38 -58 21 2 28 9 30 32 5 45 -47 65 -68 26z"/>
+ <path d="M10651 1703 c-56 -59 -101 -113 -101 -120 0 -28 172 -103 237 -103 l26 0 -7 123 c-10 179 -15 207 -36 207 -10 0 -63 -48 -119 -107z"/>
+ <path d="M11035 1801 c-7 -12 -23 -144 -29 -243 -4 -77 -4 -78 19 -78 45 0 130 22 193 51 l64 29 -19 23 c-65 82 -198 227 -209 227 -7 0 -15 -4 -19 -9z"/>
+ </g>
+ </svg>`)
+         : renderColor(winner.color)
+ }</td>
+ <td>${winner.name === undefined ? (winner.name = 'Tesla') : winner.name}</td>
+ <td>${winner.wins}</td>
+ <td>${winner.time}</td>
+ </tr>
+ `
+     )
+     .join('')}
+ </tbody>
+ </table>
+ `;
+
+export const render = async () => {
+    const html = `
+	<div class="menu">
+		<button class="to-garage" id="to-garage">To garage</button>
+		<button class="to-win" id="to-win">To winners</button>
+	</div>
+	<div class="garage-catalog">
+		<form class="form" id="create">
+			<input type="text" class="namecar" name="name" id="createname">
+			<input type="color" class="color" name="color" value="#ffffff" id="createcolor">
+			<button class="button" type="submit"> Create</button>
+		</form>
+		<form class="form" id="up">
+			<input type="text" class="upnamecar" name="upname" id="upname" disabled>
+			<input type="color" class="upcolor" name="color" value="#ffffff" id="upcolor" disabled>
+			<button class="button-up" type="submit" id="upd" disabled> Update</button>
+		</form>
+
+		<div class="controls">
+			<button class="race" id="race">Race</button>
+			<button class="reset" id="reset ">Reset</button>
+			<button class="generator" id="generator ">Generate cars</button>
+		</div>
+		</div>
+		<div id="winners" class="winners">
+		${renderWinners()}
+		</div>
+		<div id="mes" class="mes">
+
+</div>
+		<div id="garage">
+			${renderGarage()}
+		</div>
+		<div class="pag">
+<button class="prev" id="prev" disabled>Prev</button>
+<button class="next" id="next"  disabled>Next</button>
+</div>
+`;
+
+    const carHTML = document.createElement('div');
+    carHTML.innerHTML = html;
+    document.body.insertAdjacentElement('afterbegin', carHTML);
+};
+
+export const updateGarage = async (): Promise<void> => {
+    const { items, count } = await getCars(str.carsPage);
+    str.cars = items;
+    str.carsCount = count;
+    if (str.carsPage * 7 < Number(str.carsCount)) {
```

Comment:
The number of items per page is better to store in a configuration somewhere. Direct usage in multiple places makes it complex to support